server/views/index.jade

```diff
@@ -16,10 +16,7 @@ block content
             p.help= t('index noscript message')
 
 block js
-    script(src="/scripts/vendor.js")
     if env
         script.
-            require.register('env', function(exports, require, module) {
-              module.exports = !{JSON.stringify(env)}
-            })
-    script(src="/scripts/app.js", onload="require('initialize');")
+            ENV = !{JSON.stringify(env)};
+    script(src="/#{assets? assets.js : 'app.js'}" defer=true)
```

Comment:
It's already the case in `initialiaze` when we try to load the `webpack-assets.json` file. Do you prefer we define a fallback explicitely instead of in templates?